roles/rhol_crc/tasks/configuration.yml

```diff
@@ -16,21 +16,42 @@
 
 
 - name: Set RHOL/CRC configuration options and if fail print
-  block:
-    - name: Set RHOL/CRC configuration options
-      ansible.builtin.shell:  # noqa risky-shell-pipe
-        cmd: >-
-          {{ cifmw_rhol_crc_binary }} config set
-          {{ item.key }} {{ item.value }} 2>&1 |
-          tee -a {{ cifmw_rhol_crc_basedir }}/logs/crc-config.log
-      loop: >-
+  vars:
+    _patches: >-
+      {{
+        hostvars[inventory_hostname] |
+        dict2items |
+        selectattr('key', 'match', 'cifmw_rhol_crc_config_patch_.*') |
```

Comment:
I like this but I don't see where `cifmw_rhol_crc_config_patch_*` is documented.

Reply:
Ah I see, there is a typo!

Reply:
Ignore this one :)